tox.ini

```diff
@@ -1,54 +1,49 @@
 [tox]
-envlist = python
+envlist =
+    py39
+    py38
+    py37
+    py36
+    lint
 isolated_build = true
 minversion = 3.20.1
 
 [testenv]
-basepython = python3
-commands =
-    {envpython} -m \
-      pytest \
-      {posargs:}
+description = Run test suite
 deps =
     pytest
-    pytest-xdist
-isolated_build = true
-usedevelop = false
-
-[testenv:build-dists]
-description =
-    Build dists and put them into the dists/ folder
-basepython = python3
-isolated_build = true
-# `usedevelop = true` overrides `skip_install` instruction, it's unwanted
-usedevelop = false
-skip_install = true
-deps =
-    pep517 >= 0.8.2
 commands =
-    {envpython} -c 'import shutil; \
-      shutil.rmtree("{toxinidir}/dist/", ignore_errors=True)'
-    {envpython} -m pep517.build \
-      --source \
-      --binary \
-      --out-dir {toxinidir}/dist/ \
-      {toxinidir}
-
+    pytest {tty:--color=yes} {posargs:}
+package = wheel
+wheel_build_env = .pkg
```

Comment:
@gaborbernat I don't understand what these options do. They don't seem to be documented anywhere...

Reply:
I see `wheel_build_env` mentioned @ https://tox.readthedocs.io/en/rewrite/changelog.html#bugfixes-4-0-0a3 but not `package`.

Reply:
documentation is TBD... for now, consider them easter eggs

Reply:
So it's more about tox 4?